Example_Systems/PiecewiseFuel_CO2_Example/Generators_data.csv

```diff
@@ -0,0 +1,4 @@
+Resource,Zone,THERM,MUST_RUN,STOR,FLEX,HYDRO,VRE,LDS,Num_VRE_Bins,New_Build,Existing_Cap_MW,Existing_Cap_MWh,Existing_Charge_Cap_MW,Max_Cap_MW,Max_Cap_MWh,Max_Charge_Cap_MW,Min_Cap_MW,Min_Cap_MWh,Min_Charge_Cap_MW,Inv_Cost_per_MWyr,Inv_Cost_per_MWhyr,Inv_Cost_Charge_per_MWyr,Fixed_OM_Cost_per_MWyr,Fixed_OM_Cost_per_MWhyr,Fixed_OM_Cost_Charge_per_MWyr,Var_OM_Cost_per_MWh,Var_OM_Cost_per_MWh_In,Heat_Rate_MMBTU_per_MWh,Fuel,Cap_Size,Start_Cost_per_MW,Start_Fuel_MMBTU_per_MW,Up_Time,Down_Time,Ramp_Up_Percentage,Ramp_Dn_Percentage,Min_Power,Self_Disch,Eff_Up,Eff_Down,Resource_Type,region,cluster,PWFU_Slope_1,PWFU_Intercept_1,PWFU_Slope_2,PWFU_Intercept_2,CO2_Capture_Fraction,CO2_Capture_Fraction_Startup,CCS_Disposal_Cost_per_Metric_Ton,Biomass
+onshore_wind,1,0,0,0,0,0,1,0,1,0,15000,0,0,-1,-1,-1,0,0,0,0,0,0,0,0,0,0,0,0,None,100,0,0,0,0,1,1,0,0,1,1,onshore_wind_turbine,NE,1,0,0,0,0,0,0,0,0
+natural_gas_combined_cycle_ccs,1,1,0,0,0,0,0,0,0,0,15000,0,0,-1,-1,-1,0,0,0,0,0,0,0,0,0,5,0,8,NG,250,91,2,6,6,0.64,0.64,0.468,0,1,1,natural_gas_fired_combined_cycle_ccs,NE,1,6,400,7.2,200,0.9,0.6,20,0
+biomass_ccs,1,1,0,0,0,0,0,0,0,0,300,0,0,-1,-1,-1,0,0,0,0,0,0,0,0,0,10,0,10,Biomass,300,91,2,6,6,0.64,0.64,0.468,0,1,1,biomass_ccs,NE,1,0,0,0,0,0.9,0.6,20,1
```

Comment:
Why does `natural_gas_combined_cycle_ccs` have a Heat_Rate_MMBTU_per_MWh which is positive when it also has a PWFU ?